test/TestAssets/performance/MSTest1Passing/MSTests.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace MSTestAdapterPerfTests
+{
+    [TestClass]
+	public class MSTests
```

Comment:
There is a mix of spaces and tabs, could you save the file from VS to have it fixed please?

Reply:
That file is generated by template. Let's see what the template has.

Reply:
Fixed it in the template and replaced everywhere.